app/Models/User.php

```diff
@@ -207,6 +207,13 @@ public function hasRoleWithObjectNames(string $roleName, array $objectNames)
         return false;
     }
 
+    public function scopeRole($query, $role)
```

Comment:
Are you planning to use this later, or should be removed?

Reply:
I would replace `Role::getUsers()` with the role scope everywhere. I think its the more Laravelish way to query users with certain role, and has more flexibility. You can change it easily to query for more roles efficiently, etc.
```
public function scopeRole($query, $role)
{
    $role = is_array($role) ? $role : [$role];
    return $query->whereHas('roles', function($q) use ($role) {
        $q->whereIn('name', $role);
    });
}
```

Reply:
I see, thanks for the suggestion, let's go with it.